src/operator/pairwise.ts

```diff
@@ -8,12 +8,13 @@ import {Subscriber} from '../Subscriber';
  * The (N-1)th is stored in the internal state until Nth input occurs.
  * @returns {Observable<R>} an observable of pairs of values.
  */
-export function pairwise<T>(): Observable<T> {
-  return this.lift(new PairwiseOperator());
+export function pairwise<T>(): Observable<[T, T]> {
+  let _this: Observable<T> = this;
```

Comment:
on the `self` vs `_this` vs `source`...

So far `source` has been an idiom in this library. While I realize that these all end up as members of a class, the reality is that it's only so we can compose the operators left to right.  These operators rely on almost no state at all from the instance, and under the right conditions (say the passage of the "pipe operator" `|>`), they could just be functions.

Let's stick with `source`. There isn't a compelling reason to change it.